views/js/runner/plugins/security/blurPause.js

```diff
@@ -55,6 +55,7 @@ define([
             //to know if the focus is given to inner element or the main window
             var mainFocus = true;
             var innerFocus = false;
+            var ckEditorFocus = false;
```

Comment:
no es6 yet?:-)

Reply:
es6 is not enabled for this extension yet and it out of the scope for this task